README.md

```diff
@@ -45,14 +45,16 @@ google.auth.getApplicationDefault(function(err, authClient) {
       ];
       authClient = authClient.createScoped(scopes)
     }
-    
+
     // Fetch the access token
-    authClient.getAccessToken(function(err, token)) {
+    var optionalUri = null;  // optionally specify the URI being authorized
+    authClient.getRequestMetadata(optionalUri, function(err, tokens)) {
       if (err === null) {
-        // Use the access token
+        // Use the auth tokens
+        var access_token = tokens.access_token;
```

Comment:
This is a little different from the Java and C++ implementations, which construct the bearer token and put it in the authorization header, so that the dictionary can be copied into a request directly. If we want to have something that returns an access token, perhaps it should still be called getAccessToken, and if we have getRequestMetadata it should just return a dictionary of headers.